octoprint_mrbeam/static/js/material-store.js

```diff
@@ -70,12 +71,15 @@ $(function () {
                 payload: payload,
             };
 
-            document
-                .getElementById("material_store_iframe")
-                .contentWindow.postMessage(
-                    data,
-                    self.material_store_iframe_src
-                );
+            let materialStoreIframeElement = $("#material_store_iframe");
```

Comment:
if we want to have a variable we could then also initialise it at the beginning of the method. I usually don't like to initialise jquery elements because sometimes it leads to unexpected outcomes.. For example on success of an ajax call sometimes the variable could just be empty for reasons.

Reply:
done